app/controllers/user_submissions_controller.rb

```diff
@@ -0,0 +1,16 @@
+# encoding: utf-8
+
+class UserSubmissionsController < ApplicationController
+  before_action :user?
+
+  def index
+    @query_params = params[:q] || {}
+    @query = Submission.ransack(@query_params)
+    @query.sorts = ['received asc'] if @query.sorts.empty?
+    @submissions = @query.result(distinct: true)
```

Comment:
Tutaj mamy wyświetlić tylko zgłoszenia danego użytkownika. Dlatego trzeba to zrobić następująco:
```ruby
@submissions = current_person.submissions
```
Żeby to jednak zadziałało trzeba dodać metodę:
```ruby
def current_person
  raise "User missing" unless current_user
  person = Person.find_by_email(current_user.email)
  raise "Person missing" unless person
  person
end
```
w kontrolerze `application_controller` (w sekcji private)